executable-develop-plugins.md

```
@@ -0,0 +1,365 @@
+        let json = serde_json::to_string(&resp).unwrap();
+        writeln!(out, "{}", json).unwrap();
+        out.flush().unwrap();
+    }
+}
+```
+
+---
+
+## Folder structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)